.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+    "recommendations": [
+        "salesforce.salesforcedx-vscode",
+        "redhat.vscode-xml",
+        "orta.vscode-jest",
```

Comment:
I like this plugin but find its default behavior -- to keep jest running -- quite heavy-weight. @trevor-bliss  what's your take?

Reply:
I agree that it is a bit heavy. I think it's a fine tool and I'm happy to make sure `lwc-jest` is compatible with it, but I don't feel comfortable making it an official "Salesforce recommended" extension.

I just re-enabled the extension to give it another try and it feels buggy/inconsistent. For example, if I add `console.log` statements in the test I don't see them output anywhere even though I know it's being run in watch mode. I also introduced an error to a test, and the extension successfully flagged and logged the error inline in the editor, but when I removed the error and saved the inline error message remained. I have to click the debug button again for the inline text to refresh.

Has anyone used this extension locally for a significant amount of time to be confident in it's stability?

Reply:
I'm using it all the time. Sometimes the overlay message didn't go away, which got "fixed" by doing a "Reload Window".

Personally, but that's just my taste, both are "fine, but not perfect". The other extension adds a context menu to a Jest file, so that you can run/debug jest tests.